std/http/server_test.ts

```diff
@@ -640,6 +641,25 @@ test({
   }
 });
 
+test({
+  name: "[http] close server while connection is open",
+  async fn(): Promise<void> {
+    async function iteratorReq(server: Server): Promise<void> {
+      for await (const req of server) {
+        req.respond({ body: new TextEncoder().encode(req.url) });
+      }
+    }
+
+    const server = serve(":8123");
+    iteratorReq(server);
+    // fetch keeps open connection, this when calling `server.close()`
+    // server should close all open connections
+    const res = await fetch("http://127.0.0.1:8123/hello");
```

Comment:
No longer true, rewrite it using `Deno.dial()` to present hanging of server on open connection